conduit/data/datasets/audio/ecoacoustics.py

```diff
@@ -5,17 +5,17 @@
     habitats" (Final) [Data set].
     Zenodo. https://doi.org/10.5281/zenodo.1255218
 """
-from enum import Enum, auto
+from __future__ import annotations
```

Comment:
```suggestion
```
annotations should no longer be used as it results in errors when attributes inherited by attrs classes are of types that needs to be imported

Reply:
Got it, weird, some of these changes I already have locally/on my fork.